src/Components/ReviewFormDetails.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+
+const ReviewFormDetails = () => {
+    return (
+    <div>
+        <header>
+           <h2 className="form-heading">Enter the Juicy Details Here! </h2>
+         </header>
+         <div className="form-container">
+             <p>Enter required information here:</p>
+            <input
+                id="review"
+                type="text"
+                name="review"
+                placeholder="Short Review"
+                required
+                // value=""
+                // onChange={event => this.handleChange(event)}
```

Comment:
Similarly to the comment above, we can use hooks to set all of these details onChange